genai-perf/genai_perf/config/input/config_endpoint.py

```diff
@@ -56,6 +59,7 @@ def __init__(self) -> None:
         self.service_kind: Any = ConfigField(
```

Comment:
If this will always be inferred then the creation of this field should be moved into the infer method and removed from parse().
You can then remove the check that looks for the user setting it, and also set the default to be `kserve` (if that simplifies the logic).

Reply:
Thanks for pointing this out! Updated.